stripe-sms-receipt/stripe_fixtures/create_customer_and_payment.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This would currently not be copied over. If you want this to be copied over you could add this as a private asset (meaning it won't be accessible outside of a Twilio Function code). You can do this by moving the file content into:
```
stripe-sms-receipt/assets/stripe_fixtures/create_customer_and_payment.private.json
```

The `.private.` is important here.

Reply:
This is only run locally via the stripe CLI, no need to be deployed.

Reply:
The problem is more that when you call `twilio serverless:init examplee --template=stripe-sms-receiept` we will not actually download everything. Instead we'll only copy what's in the subfolders `assets/` and `functions/` so your users wouldn't be able to use that local Stripe command since they would first have to download this file from GitHub manually.

Reply:
Gotcha, understood now 👍 Moved it into the assets folder and updated the README